project1/templates/user.html

```diff
@@ -18,9 +20,84 @@
         background-color: brown;
         color:white;
         }
+        

```

Comment:
@Balaji751 You can add the styles in separately in stylesheet, kept in a folder named static. It is also useful for you to know this when you use JS files. If you already know this ignore the message.